### PR TITLE
Added home button in the nav bar

### PR DESCRIPTION
**Pull request by kanishka8276**
## Description
- List out your changes
- I have added a home button to go to the index.html in the nav bar. I have kept it consistent with the theme.
- Link the Issue which this resolves
- #80 
## How has this been tested?
I have ran it in my localhost .

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

Screenshot (71)
Screenshot (72)
Screenshot (74)

### CODE REVIEW

**Review by Praful932 — CHANGES_REQUESTED** [review]
Update PR with SS/GIFs in responsive modes

**Comment by kanishka8276** [comment]
@Praful932 done!

File: templates/mainapp/layout.html [hunk 1]
```diff
@@ -35,11 +35,11 @@
 </head>
 
 <body>
-
     <header>
+	<a name='top'>
         <div id="topheader">
             <nav class="navbar navbar-expand-md navbar-light">
-                <a class="navbar-brand" href="/"><img class="navbrandimg" src="{% static 'mainapp/png/brand-img.png' %}"></a>
```
Comment (Praful932) [inline templates/mainapp/layout.html:42]: You are overiding master branch here..

**Review by Praful932 — CHANGES_REQUESTED** [review]

**Comment by kanishka8276** [comment]
@Praful932 done!

**Comment by Praful932** [comment]
@kanishka8276 There are still conflicts here.. Please resolve them

**Comment by kanishka8276** [comment]
Ok!

**Comment by kanishka8276** [comment]
@Praful932 done!

**Comment by maneprajakta** [comment]
@kanishka8276 Have you tested it locally.
This is what we can see after deploying  [here](https://kitabe-pipeline-pr-97.herokuapp.com/)
Check once.

Screenshot from 2021-03-15 14-01-11

**Comment by kanishka8276** [comment]
@maneprajakta  That's because layout.css is not committed which contained the styling for the top button.

**Comment by kanishka8276** [comment]
@Praful932 @maneprajakta  Done!
Deployment looks fine now!

**Review by Praful932 — CHANGES_REQUESTED** [review]
@kanishka8276 There are a lot of extra commits coming in your PR because your fork seems to be behind master
image

**Comment by kanishka8276** [comment]
Yes, It is behind the master. But through upriver it's not updating . I tried many times.

**Comment by kanishka8276** [comment]
I have updated my fork. Now what should I do?

**Comment by kanishka8276** [comment]
> @kanishka8276 There are a lot of extra commits coming in your PR because your fork seems to be behind master
> image

@Praful932  I have updated my fork. Now, what should I do?

**Comment by Praful932** [comment]
@kanishka8276 You need to either rebase/merge with master from your fork, there's a lot of extra commits coming in..

**Comment by kanishka8276** [comment]
> @kanishka8276 You need to either rebase/merge with master from your fork, there's a lot of extra commits coming in..

Shall I make a new branch after pulling again from the fork and make changes.

**Comment by Praful932** [comment]
> > @kanishka8276 You need to either rebase/merge with master from your fork, there's a lot of extra commits coming in..
> 
> Shall I make a PR from a new branch after pulling again from the fork and make changes.

That works, close the PR, and make a new branch from master

**Comment by kanishka8276** [comment]
@Praful932  I have made another PR
#102 
Please review it.
And it's a humble request please deploy it .
I haven't uploaded the ss but I think the code should work fine
I have edited the file in the browser itself because through git it was including all the extra commits as well, I used the git pull command after updating the fork. But it didn't work.

**Pull request closed without merging**